src/frontend/overlay/src/components/organisms/widgets/Statistics.tsx

```diff
@@ -37,7 +37,9 @@ const Title = styled.div`
   font-weight: ${c.GLOBAL_DEFAULT_FONT_WEIGHT_BOLD};
 `;
 
-const Caption = styled.div``;
+const Caption = styled.div`
+    width: 100%;
```

Comment:
Use flex flow to ajust the desired positions.
Flex-grow would be best.
If needed, override styles of StatisticsLegend so that it doesn't take up that much space.
You will need to pass classNames for https://github.com/icpc/live-v3/blob/d85a07815acadbaab49b14fd020202daf08207b6/src/frontend/overlay/src/components/molecules/statistics/StatisticsLegend.tsx#L48

Reply:
I used flex flow in Header. Also I used flex grow in Caption and LegendsWrapper